src/BloomExe/CollectionTab/LibraryListView.cs

```diff
@@ -1210,4 +1231,27 @@ private static void CopyDirectory(string sourceDir, string targetDir)
 				CopyDirectory(directory, Path.Combine(targetDir, Path.GetFileName(directory)));
 		}
 	}
+
+	internal class BookButtonInfo
+	{
+		private readonly BookInfo _bookInfo;
+		internal BookInfo BookInfo { get { return _bookInfo; } }
+
+		private readonly BookCollection _collection;
+
+		// isVernacular meaning part of the current TheOneEditableCollection
+		private readonly bool _isVernacularBook;
```

Comment:
Don't like the name much. If someone is authoring a template in English, I would not think of its as a 'vernacular' book. IsIn{TheOne}EditableCollection?